RAMP/disposableredis/__init__.py

```diff
@@ -41,6 +41,22 @@ def __init__(self, port=None, path='redis-server', verbose=False, **extra_args):
                ))
         self.path = os.getenv(REDIS_PATH_ENVVAR, path)
 
+    def _getRedisVersion(self):
+        options = {
+            'stderr': subprocess.PIPE,
+            'stdin': subprocess.PIPE,
+            'stdout': subprocess.PIPE,
+        }
+        p = subprocess.Popen(args=[self.path, '--version'], **options)
```

Comment:
You can simplify all this with [subprocess.Run](https://docs.python.org/3/library/subprocess.html#subprocess.run). Check out the capture_output and check args.

Reply:
It was already like this and this PR did not change, I prefer, that if we do this change, it will be on separate PR.